README.md

```diff
@@ -82,9 +82,8 @@ DBLINK() uses ODBC to interact with the remote databases so you will have to ins
 - configure the ODBC layer (see the examples here below)
 
 ### Install DBLINK()
-- First you need to [Setup a C++ Development Environment](https://www.vertica.com/docs/11.1.x/HTML/Content/Authoring/ExtendingVertica/C++/C++SDK.htm) 
-- Then you will have to compile the DBLINK source code: ```make```
-- Deploy the library in Vertica (as dbadmin): ```make deploy```
+- First you need to compile the DBLINK source code for the appropriate vertica version and linux distribution: ```make VERTICA_VERSION=12.0.2 OSTAG=ubuntu```
```

Comment:
I think these are better if they arenumbered 1-3 bc they are steps. Below are a few edits that I think we should make:

1. Compile the DBLINK source code with `make` and the appropriate Vertica version and Linux distribution. For example, ```make VERTICA_VERSION=12.0.2 OSTAG=ubuntu```.
2. Deploy the library in Vertica as dbadmin with ```make install```.
3. Create a ...

Reply:
Done.  Viewable here: https://github.com/NerdLogic/dblink/tree/duane/build_with_container

Reply:
Looks good. I think we should capitalize Vertica and Linux in the first step too

Reply:
makefile with VERTICA_VERSION and OSTAG looks very good. 

What about Vertica SDK requirements and possible conflicts with standard gcc/g++? For example the current Ubuntu Long Term Support version 22.04 ships by default gcc/g++ version 11 which is not compatible with the current Vertica SDK. In this case you will have to install gcc-9 (alongside gcc-11) and manage the different alternatives. Do we want to manage (or say something about) these things?

Reply:
???
g++ v11 is compatible with Vertica, right?  In fact, I thought v11 was a requirement.
The docker containers are built with ubuntu 18.04, so it should still build.  There are many ways of setting up gcc that its much easier to have the whole environment inside a container.
Do you mean that the .so file will not be compatible with Vertica running on 22.04?

Reply:
GNU CC/C++ version 11 (which is different from the standard ISO/IEC 14882 released in 2011 and known as C++11) is not compatible with Vertica SDK...
```
g++ -O3 -D HAVE_LONG_INT_64 -Wall -std=c++11 -shared -Wno-unused-value -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC  -I/opt/vertica/sdk/include -I/opt/vertica/sdk/examples/HelperLibraries -o /tmp/ldblink.so ldblink.cpp /opt/vertica/sdk/include/Vertica.cpp -lodbc
In file included from /opt/vertica/sdk/include/Vertica.h:91,
                 from ldblink.cpp:2:
/opt/vertica/sdk/include/BuildAssertions.h:80:10: error: #error "Vertica does not build on GCC v11.0 or above."
   80 |         #error "Vertica does not build on GCC v11.0 or above."
      |          ^~~~~
...
```
So the following are both true:
1. using `-std=c++11` to specify ISO/IEC 14882 released in 2011 is a must
2. Vertica SDK does not build with g++ v11 (GNU c++ version 11)

Reply:
> The docker containers are built with ubuntu 18.04, so it should still build

Right. Ubuntu LTS 18.04 ships by default with g++ v7 so we are ok for the moment(*). 

(*) the maintenance window of this LTS Ubuntu 18.04 will expire April 2023

Reply:
Thanks for explaining that.  I was confused about the two version-11s.  I don't think it has any specific implications for this PR, but thanks for bringing this to my attention.